content/docs/azure/authentication.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Restricting Access & Authentication "
+description = "Restrict access of your deployment to specified IP addresses"
+weight = 100
++++
+
+This section shows how to restrict access to only certain IP addresses for your LoadBalancer Service. At a later date, it will also include formal authentication through Azure. This method is not the most ideal way to secure your Kubernetes cluster, as it requires that you access the service from the same IP address every time. This process was adapted from [this guide](https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service).
```

Comment:
Recommendation: It's a good idea to give people a hint about where a link will take them, especially if the link goes outside the current doc set. Change `[this guide]` to something like `[the Kubernetes guide to configuring a firewall]` or whatever words best describe the doc publisher and topic.